circe/src/main/scala/org/http4s/circe/CirceInstances.scala

```diff
@@ -18,14 +20,14 @@ trait CirceInstances {
       )
     }
 
-  implicit val jsonEncoder: EntityEncoder[Json] =
-    EntityEncoder[String].contramap[Json] { json =>
-      // Comment from ArgonautInstances (which this code is based on):
-      // TODO naive implementation materializes to a String.
-      // See https://github.com/non/jawn/issues/6#issuecomment-65018736
-      Printer.noSpaces.pretty(json)
-    }.withContentType(`Content-Type`(MediaType.`application/json`))
+  implicit def jsonEncoder: EntityEncoder[Json] =
```

Comment:
Can it be made
```
implicit def jsonEncoder(implicit printer: Printer = Printer.noSpaces): EntityEncoder[Json] = ...
```
Then you can pass another implicit printer in scope where you use the `jsonEncoder` to override the default one.